### ISSUE TITLE
Comune di Pisa: problemi di integrazione

### ISSUE THREAD

**Issue by sirio1970** [italia/spid#1211]
Salve, per il comune di Pisa abbiamo attivato già da molto tempo l'integrazione con SPID. Da ieri abbiamo problemi con tutti gli idp ad esclusione di Lepida.
 L'errore ricevuto in genere è il seguente:
 Impossibile stabilire l'autenticità della richiesta di autenticazione - Contattare il gestore del servizio
 Sapete inviarci qualche dettaglio per risolvere il problema?
 Grazie
 sirio

**Comment by sirio1970** [comment]
Buongiorno, avete novità in merito alla segnalazione? Arrivano molte segnalazioni da parte degli utenti del Comune di Pisa.
Grazie
sirio

**Comment by Gnafu** [comment]
Buongiorno, questa segnalazione è collegata a #1217 ?

**Comment by sirio1970** [comment]
Esatto. Mi sono accorto che il certificato era scaduto. Riuscite a pubblicare i metadata che il comune è su tutte le furie?


Grazie

sirio

**Comment by Gnafu** [comment]
Credo debbano occuparsene @umbros , @AntonioGiovanniSchiavone o @damikael ma non so quanto tempo ci voglia.
Anch'io sto aspettando di poter accedere tramite SPID. Nel frattempo cercherò di usare un login diverso.
